lib/Dialect/MSFT/ExportQuartusTcl.cpp

```diff
@@ -332,17 +333,12 @@ LogicalResult TclEmitter::emit(Operation *hwMod, StringRef outputFile) {
   }
 
   // Create a verbatim op containing the Tcl and symbol references.
-  OpBuilder builder = OpBuilder::atBlockEnd(hwMod->getBlock());
-  auto verbatim = builder.create<sv::VerbatimOp>(
-      builder.getUnknownLoc(), os.str(), ValueRange{},
-      builder.getArrayAttr(state.symbolRefs));
-
-  // When requested, give the verbatim op an output file.
-  if (!outputFile.empty()) {
-    auto outputFileAttr =
-        OutputFileAttr::getFromFilename(builder.getContext(), outputFile);
-    verbatim->setAttr("output_file", outputFileAttr);
-  }
+  auto builder = ImplicitLocOpBuilder::atBlockEnd(
+      UnknownLoc::get(hwMod->getContext()), hwMod->getBlock());
+  builder.create<emit::FileOp>(outputFile, [&] {
```

Comment:
Probably this is correct, but does the empty string here do what's intended?

Reply:
If it's okay with the MSFT folks, I'll add a check to bail out if the filename is empty.